src/app/copy/inline-copy/inline-copy.component.less

```diff
@@ -0,0 +1,31 @@
+.inline-copy {
+  display: inline-flex;
+  border: 1px solid #bbb;
+  background-color: #fff;
+  font-size: smaller;
+  margin-left: .25em;
+  margin-right: .25em;
+  max-width: 100%;
+
+  .token-cont {
```

Comment:
Thinking we should chose a more generic selector name? I imagine this component will be used to copy other values besides a token.

In the block-copy css, it looks like we're using `txt-cont`?